### DISCUSSION TITLE
Pose estimator with flask example: Unable to get key points on live stream to detect the pose.

### DISCUSSION THREAD

**Discussion by hardikec2023** [AxisCommunications/acap-computer-vision-sdk-examples#150] [Q&A] [answered]
We have successfully deployed pose estimator with flask example on ARTPEC 8 camera but we are not able to detect pose on live stream. We are getting result on terminal but the inference server at http://$DEVICE_IP:5000 is not showing the key points on live stream.

**Comment by Corallo** [comment, marked as answer]
This issue is already reported [here](https://github.com/AxisCommunications/acap-computer-vision-sdk-examples/issues/148)
